include/exadg/time_integration/time_int_explicit_runge_kutta_base.cpp

```diff
@@ -162,28 +157,17 @@ TimeIntExplRKBase<Number>::do_read_restart(std::ifstream & in)
 
   // Note that the operations done here must be in sync with the output.
 
-  // 1. ranks
```

Comment:
note that we do not have th constraint
`n_ranks_old == n_ranks`
anymore because we use deal.II's serialization now!